templates/kubernetes/what-is-kubernetes.html

```diff
@@ -284,226 +305,339 @@ <h2 class="p-muted-heading u-align--center">
-    <div class="row">
-      <div class="col-12">
-        <h2>
-          Why use Kubernetes?
-        </h2>
-        <p>
-          Kubernetes is a platform to run your applications and services. It's cloud-native, provides operational cost savings, faster time-to-market and is maintained by a large community. Developers like container-based development, as it helps break up monolithic applications into more maintainable microservices. Kubernetes allows their work to move seamlessly from development to production and results in faster-time-to-market for a businesses' applications.
-        </p>
-        <p>
-          Kubernetes works by:
-        </p>
-        <ul class="p-list is-split">
-          <li class="p-list__item is-ticked">Orchestrating containers across multiple hosts
+      <aside class="p-accordion">
+        <ul class="p-accordion__list">
+          <li class="p-accordion__group">
+            <div type="button" role="heading" aria-level="3" class="p-accordion__heading">
+              <button class="p-accordion__tab" id="tab1" aria-controls="tab1-section" aria-expanded="true">
+                Node
+              </button>
+            </div>
+            <section class="p-accordion__panel" id="tab1-section" aria-hidden="false" aria-labelledby="tab1">
+              <p>
+                A virtual or physical machine, depending on the cluster setup. Each node is managed by the control plane and contains the services necessary to run pods.
+              </p>
+            </section>
           </li>
-          <li class="p-list__item is-ticked">Ensures that containerised apps behave in the same way in all environments, from testing to production
+          <li class="p-accordion__group">
+            <div>
+              <button type="button" class="p-accordion__tab" id="tab2" aria-controls="tab2-section" aria-expanded="true">
+                Pod
+              </button>
+            </div>
+            <section class="p-accordion__panel" id="tab2-section" aria-hidden="false" aria-labelledby="tab2">
+              <p>
+                The smallest unit in the Kubernetes object model that is used to host containers.
+              </p>
+            </section>
           </li>
-          <li class="p-list__item is-ticked">Controlling and automating application deployments and updates
+          <li class="p-accordion__group">
+            <div>
+              <button type="button" class="p-accordion__tab" id="tab3" aria-controls="tab3-section" aria-expanded="true">
+                Container
+              </button>
+            </div>
+            <section class="p-accordion__panel" id="tab3-section" aria-hidden="false" aria-labelledby="tab3">
+              <p>
+                A standard unit of software that packages up code and all its dependencies, making it portable across different compute environments. <a href="/containers/what-are-containers" class="p-link--external">Learn more about containers</a>
```

Comment:
The link goes to a 404 page!

Reply:
Ah yup, it's because the page doesn't exist yet. Will take it out and leave a comment on the copy doc!